src/fuse/utils.rs

```diff
@@ -0,0 +1,76 @@
+macro_rules! impl_from {
```

Comment:
change the macro name to a more meaningful one

Reply:
Done.